wpilib/wpilib/smartdashboard.py

```diff
@@ -378,7 +379,7 @@ def setDefaultBooleanArray(cls, key: str, defaultValue: List[bool]) -> bool:
         return table.setDefaultBooleanArray(key, defaultValue)
 
     @classmethod
-    def getBooleanArray(cls, key: str, defaultValue: List[bool]) -> List[bool]:
+    def getBooleanArray(cls, key: str, defaultValue: Iterable[bool]) -> Iterable[bool]:
```

Comment:
I don't think this is the contract we want to give to teams. This seems bad.

Reply:
defaultValue *is* of type iterable (ex. https://github.com/robotpy/pynetworktables/blob/master/networktables/networktable.py#L668), and it seems that we always return a Tuple, unless the key isn't found in which case we return the defaultValue (which is iterable)

Reply:
Bleh. I guess we should fix those. Would you agree that being able to index an array value is useful?

Reply:
mmmm yeah I do. I'll change it to sequence here since it isn't that important anyway

Reply:
As an aside, I think specifying _any_ arbitrary default value should be legal. Imagine a team that wants to handle missing values specially, say, having a default that is expensive to compute, so they pass in None and check whether the get returns None.